app/lib/activitypub/activity/create.rb

```diff
@@ -490,7 +490,16 @@ def tombstone_exists?
   def forward_for_reply
     return unless @status.distributable? && @json['signature'].present? && reply_to_local?
 
-    ActivityPub::RawDistributionWorker.perform_async(Oj.dump(@json), replied_to_status.account_id, [@account.preferred_inbox_url])
+    # If the status is replying to a local status, we should forward it
+    # everywhere where that local status was distributed
+
+    status_reach_finder = StatusReachFinder.new(@status)
```

Comment:
Am I wrong in understanding that the computed reach includes, for instance, people mentioned by this status? Meaning you'd re-distribute to remote users who were explicitly mentioned by the remote author?